backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/invoice/isCompleteInvoice.feature

```diff
@@ -0,0 +1,120 @@
+@from:cucumber
+@dev:runThisOne
+Feature: completeInvoice option when processing invoice candidates
+
+  Background:
+    Given the existing user with login 'metasfresh' receives a random a API token for the existing role with name 'WebUI'
+    And set sys config boolean value true for sys config SKIP_WP_PROCESSOR_FOR_AUTOMATION
+    And set sys config boolean value true for sys config de.metas.report.jasper.IsMockReportService
+    And metasfresh has date and time 2021-12-21T13:30:13+01:00[Europe/Berlin]
+
+  @from:cucumber
+  Scenario: process IC with completeInvoices=N should result in Invoice docstatus=IP and IC should be invoiceable again after voiding Invoice
+    Given metasfresh contains M_Products:
+      | Identifier | Name              |
+      | p_ci_1     | salesProduct_ci_1 |
+    And metasfresh contains M_PricingSystems
+      | Identifier | Name                | Value                | OPT.Description            | OPT.IsActive |
+      | ps_ci_1    | pricing_system_name | pricing_system_value | pricing_system_description | true         |
+    And metasfresh contains M_PriceLists
+      | Identifier | M_PricingSystem_ID.Identifier | OPT.C_Country.CountryCode | C_Currency.ISO_Code | Name            | OPT.Description | SOTrx | IsTaxIncluded | PricePrecision | OPT.IsActive |
+      | pl_ci_1    | ps_ci_1                       | DE                        | EUR                 | price_list_name | null            | true  | false         | 2              | true         |
+    And metasfresh contains M_PriceList_Versions
+      | Identifier | M_PriceList_ID.Identifier | Name           | ValidFrom  |
+      | plv_ci_1   | pl_ci_1                   | salesOrder-PLV | 2021-04-01 |
+    And metasfresh contains M_ProductPrices
+      | Identifier | M_PriceList_Version_ID.Identifier | M_Product_ID.Identifier | PriceStd | C_UOM_ID.X12DE355 | C_TaxCategory_ID.InternalName |
+      | pp_ci_1    | plv_ci_1                          | p_ci_1                  | 10.0     | PCE               | Normal                        |
+    And metasfresh contains C_BPartners:
+      | Identifier       | Name        | OPT.IsVendor | OPT.IsCustomer | M_PricingSystem_ID.Identifier |
+      | endcustomer_ci_1 | Endcustomer | N            | Y              | ps_ci_1                       |
+    And metasfresh contains C_BPartner_Locations:
+      | Identifier | GLN           | C_BPartner_ID.Identifier | OPT.IsShipToDefault | OPT.IsBillToDefault |
+      | l_ci_1     | 0123456789011 | endcustomer_ci_1         | Y                   | Y                   |
+    And metasfresh contains C_Orders:
+      | Identifier | IsSOTrx | C_BPartner_ID.Identifier | DateOrdered | OPT.POReference  | OPT.C_PaymentTerm_ID |
```

Comment:
AFAIU setting the C_PaymentTerm_ID is not needed
LE: ..but OK with me - after all you later verify that the payment-term is propagated